### ISSUE TITLE
Only view file with onlyoffice plugin

### ISSUE THREAD

**Issue by Bust011r** [humhub/humhub#5519]
Hi,

is it possible disable the download of the file and have only onlyoffice view mode enabled?

**Comment by luke-** [comment]
No, presumably it would be downloadable via OnlyOffice anyway, if you want to protect it.
